pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -115,15 +115,22 @@ func (r *ReconcileCincinnati) Reconcile(request reconcile.Request) (reconcile.Re
 	instanceCopy := instance.DeepCopy()
 	instanceCopy.Status = cv1alpha1.CincinnatiStatus{}
 
-	for _, f := range []func(context.Context, logr.Logger, *cv1alpha1.Cincinnati) error{
+	// this object creates all the kube resources we need and then holds them as
+	// the canonical reference for those resources during reconciliation.
+	resources, err := newKubeResources(instanceCopy, r.operandImage)
```

Comment:
this feels like the kind of error that would be pretty catastrophic. I think we should log it.